policy/modules/admin/usermanage.te

```diff
@@ -500,6 +501,7 @@ init_dontaudit_rw_utmp(sysadm_passwd_t)
 
 logging_send_syslog_msg(sysadm_passwd_t)
 
+userdom_manage_admin_files(sysadm_passwd_t)
```

Comment:
This one should IMHO go away too.

Reply:
Do you know about other way how to allow access? I have created file transition from admin_home_t to passwd_file_t and shadow_t, but AVC's messages are still appears.

Reply:
@zpytela

Reply:
I am okay with vipw not writing to .viminfo.